spyderlib/otherplugins.py

```diff
@@ -8,80 +8,69 @@
 Spyder third-party plugins configuration management
 """
 
-import importlib
 import os
 import os.path as osp
 import sys
 import traceback
 
 # Local imports
 from spyderlib.config.base import get_conf_path
-from spyderlib.py3compat import PY2, PY3, PY33
+from spyderlib.py3compat import PY2
 
 if PY2:
     import imp
+else:
+    import importlib
 
 
-def _get_spyderplugins(plugin_path, base_namespace, plugins_namespace,
-                       modnames, modlist):
-    """Scan the directory `plugin_path` for plugins_namespace package and
-    loads its submodules."""
-    namespace_path = osp.join(plugin_path, base_namespace, plugins_namespace)
+PLUGIN_PREFIX = "spyder_"
+IO_PREFIX = PLUGIN_PREFIX + "io_"

```

Comment:
Question here: What's the name for `io` plugins according to this? Is it `spyder_io_foo` or `spyderio_foo`?

I say that because you named our io plugins like `spyderio_foo`, so those names or this line should be changed :-)